src/content/documentation/develop/developing-extensions-for-firefox-for-android.md

```diff
@@ -143,59 +151,17 @@ In your extension, ensure that you've included an application ID using the [brow
 }
 ```
 
-If you’re using web-ext, follow the [Testing in Firefox for Android](/documentation/develop/getting-started-with-web-ext#testing-in-firefox-for-android) instructions.
-
-Otherwise, [zip the content of your extension into an xpi file](/documentation/publish/package-your-extension) named to match the application ID, for example, `borderify@example.com.xpi`.
-
-You now have two options for transferring and running your extension: using adb or a website.
-
-### Transfer your extension using adb
-
-On your computer, execute `/path/to/adb push /path/to/<extension file name>.xpi /mnt/sdcard/`, which will transfer the extensions xpi file to your attached emulator or device.
-
-On your Android device or in the emulator, open Firefox for Android and browse to `file:///mnt/sdcard`:
-
-<article class="module-content grid-x grid-padding-x">
-<div class="cell small-6">
+In the unzipped directory of your extension, run `web-ext run -t firefox-android` and follow the instructions on screen to make sure you select the right device. Select `org.mozilla.fenix` as the apkname.
 
-![xpi file on memory card](/assets/img/documentation/develop/xpi-file-on-memory-card.png)
+Here is an example of the command:
 
-</div>
-</article>
+<!-- Syntax Highlighting -->
 
-Tap on `<extension file name>.xpi` to install it. You will get a warning about the extension being blocked, tap ALLOW:
-
-<article class="module-content grid-x grid-padding-x">
-<div class="cell small-6">
-
-![Blocked add-on message](/assets/img/documentation/develop/blocked-add-on-message.png)
-
-</div>
-</article>
-
-An additional warning will tell you the extension is unverified, tap INSTALL:
-
-<article class="module-content grid-x grid-padding-x">
-<div class="cell small-6">
-
-![Unverified add-on message](/assets/img/documentation/develop/unverified-add-on-messages.png)
-
-</div>
-</article>
-
-Your extension will start running (in this case a copy of the [borderify](https://developer.mozilla.org/Add-ons/WebExtensions/Examples#borderify) example extension):
-
-<article class="module-content grid-x grid-padding-x">
-<div class="cell small-6">
-
-![Borderify sample extension in action](/assets/img/documentation/develop/borderify-in-action.png)
-
-</div>
-</article>
-
-### Transfer your extension via a website
+```shell
+web-ext run -t firefox-android --adb-device XXX --firefox-apk org.mozilla.fenix
+```
 
-Upload your xpi file to your website and make it accessible over HTTP. Browse to the file and download it. Follow the installation instructions, which will be similar to those for an extension transferred using adb.
+<!-- END: Syntax Highlighting -->
 
 {% endcapture %}
 {% include modules/one-column.liquid
```

Comment:
@caitmuenster as we agreed on Friday last week I double-checked the content of the section **"Debug you extension"**, a good chunk of the content can stay as it is, but there are some changes needed:

- add a note to explicitly mention that in Fenix the extensions are enabled lazily once at least one tab has been opened (mainly because it may surprise some user that none of the extensions code is loaded if they connect to the debugger before any tab has been actually opened)

- There is currently a Fenix-specific issue with using the devtools inspector on targeting the Fenix's browserActor popups ([Bug 1637616](https://bugzilla.mozilla.org/1637616)), it may be good to add a "warning" note at the start of the **Debug your extension** to point out this issue to the developers (which is unfortunately quite annoying, as a workaround a developer may temporarily change the extension to open the popup extension page into a tab to be able to inspect it using the developer tools) 

- we may change a bit the note at line 204-206 (`Unlike desktop Firefox, where content scripts are debugged in`), in my opinion we may just remove the `"Unlike desktop Firefox, ..."` and just say something like `"Content scripts can be reached and debugged when connected to the main process or to a tab target where a content script is running"` (the reason for this change is that in the meantime the devtools team has introduced multiprocess support to the Browser Toolbox and so the "reason why" this works is not the same as it was for Fennec and it is not actually even specific to mobile)

We may leave the rest of the content unmodified for now (there is currently no mention to connecting from about:debugging to a specific extension running on Fenix, which does actually also work).

Reply:
Thanks @rpl! I just pushed a new commit to make those changes. If you could re-review that would be great!